.github/workflows/deploy-release-reusable.yaml

```diff
@@ -264,18 +280,16 @@ jobs:
           # Checkout the SVN repository
           export SVN_DIR="/tmp/svn-repo"
           svn co \
-            --depth files \
-            "https://dist.apache.org/repos/dist/dev/logging" \
+            "https://dist.apache.org/repos/dist/dev/logging/$PROJECT_ID" \
```

Comment:
@ppkarwasz, note the assumption here: `PROJECT_ID` folder (i.e., `logging-parent`, `log4j`, `log4jt-tools`, etc.) must exist. The amount of boilerplate needed to deal with the intricacies of Subversion, and caching, and whatnot is frustrating; hence this simplification assumption.

Reply:
I don't think this is a problem.